packages/sit-onyx/src/components/OnyxTextarea/OnyxTextarea.stories.ts

```diff
@@ -138,3 +138,30 @@ export const Autosize = {
     },
   },
 } satisfies Story;
+
+/**
+ * This example shows a textarea with label tooltip.
+ */
+export const WithLabelTooltip: Story = {
+  args: {
+    label: "Label",
+    labelTooltip: "More Information",
+  },
+  decorators: [
+    (story) => ({
+      components: { story },
+      template: `<div style="padding-top: 2rem"> <story /> </div>`,
+    }),
+  ],
+};
```

Comment:
```suggestion
export const WithLabelTooltip = {
  args: {
    label: "Label",
    labelTooltip: "More Information",
  },
  decorators: [
    (story) => ({
      components: { story },
      template: `<div style="padding-top: 2rem"> <story /> </div>`,
    }),
  ],
} satisfies Story;
```

We usually use `satisfies Story` for better type intellisense

Reply:
An error shows up 
![image](https://github.com/SchwarzIT/onyx/assets/60856270/164a4e89-f773-462c-837d-d419fd701ff1)

That was the reason why I did it this way